hilti/src/ast/builder/type.cc

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2020 by the Zeek Project. See LICENSE for details.
+
+#include <hilti/ast/ctors/list.h>
+#include <hilti/ast/expression.h>
+#include <hilti/ast/types/unknown.h>
+
+using namespace hilti;
+
+Type builder::typeOfExpressions(const std::vector<Expression>& e) {
```

Comment:
I am surprised this actually implements the correct function 😅 . Maybe instead do the following (requires manual adjustment at the end of the namespace).
```suggestion
namespace hilti::builder {
Type typeOfExpressions(const std::vector<Expression>& e) {
```

Reply:
This pattern is used quite a bit across the code (the *.cc files usually don't open a namespace), so I'll leave as is for consistency. It as some point we wanted to do a bulk change, I wouldn't object.